### ISSUE TITLE
References do not load

### ISSUE THREAD

**Issue by nickcoad** [Sitefinity/Sitefinity-External-STS-Integration#13]
When I add this project to my Sitefinity website project (by following the instructions in the wiki) none of the references load.  I've attached an image of what I see in the references list immediately after following the instructions.

![brokenrefs](https://cloud.githubusercontent.com/assets/2467917/9080109/fcc31a56-3b93-11e5-8f87-5e72e7810b98.png)

To reproduce, simply follow the instructions on the wiki.

**Comment by georgievrado** [comment]
Did you restore nuget packages?

**Comment by nickcoad** [comment]
Yes, as mentioned I followed the complete list of instructions, you can reproduce the issue by doing the same. A coworker also followed the same instructions and saw the same issue.

On 5 Aug 2015 5:11 pm, Radoslav Georgiev <notifications@github.com> wrote:

Did you restore nuget packages?

**Comment by georgievrado** [comment]
I am building the project with one more extra step - unload the project and remove StyleCop targets imports and fix the Microsoft.Bcl.Build targets imports.

**Comment by nickcoad** [comment]
After following your advice I was able to get it working, but in my case I'm using a more recent Sitefinity version (8.1.5800.0) and the references the wiki lists for replacement is not the complete set.  I had to replace the following references:

Telerik.OpenAccess
Telerik.Sitefinity
Telerik.Sitefinity.Model
Telerik.Sitefinity.Personalization
Telerik.Sitefinity.Services.Comments
Telerik.Sitefinity.Services.Events
Telerik.Sitefinity.Services.Notifications
Telerik.Sitefinity.Services.RecycleBin
Telerik.Sitefinity.Services.Statistics
Telerik.Sitefinity.Utilities

Then I had a lot of warnings that were fixed with binding redirects. It looks like maybe there are a few amendments due for the wiki instructions.

Thanks a lot for your help, it's working now! :+1:
